src/serializer.tsx

```diff
@@ -115,7 +126,7 @@ export const serialize = (node: Node): string => {
  * In order to conform to slates structure, we need to create the
  * slate node such that we have: <a><em>example.com</em></a>
  */
-const normalizeMark = (node: Node, mark: string): void => {
+const normalizeMark = (node: SlateNode, mark: Mark): void => {
```

Comment:
```suggestion
const normalizeMark = (node: Exclude<SlateNode, Text>, mark: Mark): void => {
```
This can't be a text-node, because they don't have children.